py/autoevals/test_llm.py

```diff
@@ -14,7 +16,15 @@ def test_template_html():
     assert chevron.render(template_triple, dict(output="Template<Foo>")) == "Template<Foo>"
 
 
-def test_openai():
+@pytest.fixture
+def custom_client():
+    # A specific client object may be passed
+    # in lieu of automatic client creation in `LLMClassifier`
+    yield openai.OpenAI()
```

Comment:
could be nice to proxy/monkey patch this object to save a boolean that it was called when its `.chat.completions` method runs (or something like that) for the test.

Reply:
Updated with a full mock response (also has the benefit of not actually hitting OpenAI).